test/fs-writable.js

```diff
@@ -45,6 +45,15 @@ metatests.test('fs-writable / open', test => {
   test.on('done', () => fs.unlinkSync(TEST_FILENAME));
 });
 
+metatests.test('fs-writable / open with error', test => {
+  const TEST_FILENAME = __dirname;
+
+  test.plan(1);
+
+  const stream = new WritableFileStream(TEST_FILENAME);
+  stream.on('error', () => test.pass('open'));
```

Comment:
Nit: you can use
```suggestion
  stream.on('error', test.mustCall(() => test.end()));
```
Instead od `test.plan`.